{
  "uid" : "4b4757e66a1912dae1a509f688f20b0f",
  "name" : "categories",
  "children" : [ {
    "name" : "Test defects",
    "children" : [ {
      "name" : "requests.exceptions.ConnectionError: HTTPConnectionPool(host='132.232.44.158', port=5000): Max retries exceeded with url: /userLogin/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000026E311A8AC8>: Failed to establish a new connection: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。'))",
      "children" : [ {
        "name" : "test_demo3_shibai",
        "uid" : "f87d40e3875aaa32",
        "parentUid" : "df6706a80fcb442b746a67bdb4934298",
        "status" : "broken",
        "time" : {
          "start" : 1627440679805,
          "stop" : 1627440700852,
          "duration" : 21047
        },
        "flaky" : false,
        "newFailed" : false,
        "parameters" : [ ]
      } ],
      "uid" : "df6706a80fcb442b746a67bdb4934298"
    } ],
    "uid" : "bdbf199525818fae7a8651db9eafe741"
  } ]
}